workflow/drilling.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Drilling Into Your Data
```

Comment:
We should add a section on Metrics in here as well, and call Metrics as the starting point for most drills because Metrics provide the high level view of the system in question, while traces and logs provide more granular detail on what was happening.